package.json

```diff
@@ -15,13 +15,14 @@
   },
   "dependencies": {
     "axios": "^0.17.1",
-    "bootstrap": "^4.0.0",
```

Comment:
Bootstrap should be included for now (tooltip)

Reply:
keeping it out, I copied the modal style for now, will re-do it in amsterdam style in the stijl repo